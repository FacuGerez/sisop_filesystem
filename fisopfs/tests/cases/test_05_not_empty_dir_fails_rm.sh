#!/bin/bash

MOUNT=tests/mount

mkdir "$MOUNT"/testdir
touch "$MOUNT"/testdir/testfile
rmdir "$MOUNT"/testdir  2>&1
ls "$MOUNT"
ls "$MOUNT"/testdir