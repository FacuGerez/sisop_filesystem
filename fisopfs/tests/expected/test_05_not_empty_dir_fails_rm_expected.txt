rmdir: fallo al borrar 'tests/mount/testdir': El directorio no está vacío
testdir
testfile
